src/client/apiClient.ts

```diff
@@ -90,7 +90,12 @@ const postJson = <T>(
   params?: Record<string, string>
 ): Promise<T> => client.postJson<T>(buildUrl(conf, path, params), payload, headers(conf)).then(handleJsonResponse);
 
-const headers = (conf: ApiClientConfig): OutgoingHttpHeaders => ({ Authorization: conf.apiToken });
+const headers = (conf: ApiClientConfig): OutgoingHttpHeaders => ({
```

Comment:
nit: I would define them in order, and put them at top (before any usage)

Reply:
I prefer this order: exported members at the top, implementation details at the bottom. For me, that's the most logical reading order. 🤷